ansible/vars/topo_wan-pub.yml

```diff
@@ -15,6 +15,17 @@ topology:
         - 10.1.0.32/32
       ipv6:
         - FC00:1::32/128
+wan_dut_configuration:
+  vlab-01:
```

Comment:
I think hostname `vlab-01` is too generic here. Need something more generic.

Reply:
use a more generic name: WANDUT01 and add dut_offset to represent the index in inventory file.